dipy/reconst/dti.py

```diff
@@ -2050,6 +2036,117 @@ def eig_from_lo_tri(data, min_diffusivity=0):
     return dti_params.reshape(data.shape[:-1] + (12, ))
 
 
+def quantize_evecs(evecs, odf_vertices=None, v=0, nbr_processes=1):
+    """ Find the closest orientation of an evenly distributed sphere
+
+    Parameters
+    ----------
+    evecs : ndarray
+    odf_vertices : None or ndarray
+        If None, then set vertices from symmetric362 sphere.  Otherwise use
+        passed ndarray as vertices
+    v : int
+        Use v-th eigenvector as the primary eigenvector for fiber tracking.
+        If 0, uses the largest eigenvalue.
+    nbr_processes : int
+        Number of processes to use. If > 1, divides data into nbr_processes for
+        multiprocessing. If -1, uses as many processes as possible.
+
+    Returns
+    -------
+    closest : ndarray
+    """
+    if nbr_processes != 1:
```

Comment:
This will fail if `nb_processes=0` because `nbr_chunks` will be equal to 0. This is a strange case so should we manage it as 1 (not set) or -1(all cpu's).
Can you add a test for this case?